vars/emailUtils.groovy

```diff
@@ -0,0 +1,27 @@
+/**
+ * Requires "Email Notifications Plugin"
+ */
+
+private def sendEmail(sentTo, replyTo, subject, content) {
+
+        if(sentTo != null && !sentTo.isEmpty()) {
+          emailext(body: content, mimeType: 'text/html',
+             replyTo: '$DEFAULT_REPLYTO', subject: subject,
+             to: sentTo, attachLog: true )
+        }
+}
+
+
+def call(parameters = [:]) {
+        def sentTo = parameters.get('sentTo', pipelineVars.emailDefaultSentTo)
```

Comment:
To avoid confusion can we call this variable `sendTo` instead of `sentTo`

Reply:
Changed to `to` just avoid all confusion. Thanks!